lang/en_GB/server.json

```diff
@@ -268,6 +268,10 @@
 			"Poster IDs",
 			"Display thread-level IP-based poster identification mnemonics on posts"
 		],
+		"rbText": [
```

Comment:
No need to insert these manually. Just fill in the English version and run `./scripts/migrate_lang.js`. It will also preserve alphabetical ordering of keys. Run it.